suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -37,7 +37,7 @@ export type DeviceReducerState = {
     devices: TrezorDevice[];
     selectedDevice?: TrezorDevice;
     deviceAuthenticity?: Record<string, StoredAuthenticateDeviceResult>;
-    devicesWithFailedEntropyCheck?: string[];
+    devicesWithFailedEntropyCheck?: (string | null)[];
```

Comment:
What does it mean to have a `null` here? Can it be used as a device identifier?

Reply:
Protobuf allows null values here and I don't want to do `if (device.id)` because a fake device could intentionally set `id` to null.

Reply:
Added comment explaining this.